### PR TITLE
feat(jest): 42% coverage for libraries/IndexedDB AP-892

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

Self-explanatory. The reason the coverage is not higher at the moment is that I need an actual mock (and an address) to simulate the forEach loops; I am currently asking David C for it. 

**Which issue(s) this PR fixes** 🔨

AP-892

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: libraries/IndexedDB/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    db.conversations.bulkGet = jest.fn()
+
+    const result = await idb.searchMessage(
+      argumentAccounts,
+      argumentQueryOptions,
+    )
+    expect(db.conversations.bulkGet).toHaveBeenCalled()
+    console.log(result)
```
Comment (stavares843) [inline libraries/IndexedDB/index.test.ts:64]: do we need this console.log? 🔨

**Review by stavares843 — APPROVED** [review]

**Comment by stavares843** [comment]
minor comment

**Comment by WanderingHogan** [comment]
this is fine but don't add any coverage to the indexeddb work in the near term

**Review by WanderingHogan — APPROVED** [review]

**Comment by stavares843** [comment]
thanks @WanderingHogan

**Pull request merged**